app/population/views/weekly.njk

```diff
@@ -0,0 +1,22 @@
+{% extends "layouts/collection.njk" %}
+
+{% block beforeContent %}
+  <h1 class="govuk-heading-s govuk-!-margin-top-2">{{ [category, location] | join(" : ") }}</h1>
```

Comment:
This provides the "Category : Location" that appears below the Organization switcher. 

Ideally I think this should live inside a version of that component, as having category and location visible when within a region is very useful - however, I'm not 100% sure if the current getLocation call includes the category information, and thus is available on every single page.

Reply:
Will the position of this set above the breadcrumb given the `super()` call is after this line?

Reply:
I also have a few other questions about this element, it might be easier jumping on a call when you've got a minute?

Reply:
Discussed this on a video call - decided to remove this feature until we can talk about it with Andy. In the meantime there is enough context on the various pages for the user to know what category and location they are working with.